incubator/binding-openapi/src/main/java/io/aklivity/zilla/runtime/binding/openapi/internal/config/OpenapiCompositeBindingAdapter.java

```diff
@@ -0,0 +1,798 @@
+    {
+        if (isJwtEnabled)
+        {
+            namespace
+                .guard()
+                    .name("jwt0")
+                    .type("jwt")
+                    .options(JwtOptionsConfig::builder)
+                        .issuer("") // env
+                        .audience("") // env
+                        .key()
+                            .alg("").kty("").kid("").use("").n("").e("").crv("").x("").y("") // env
+                            .build()
+                        .build()
+                    .build();
+        }
+        return namespace;
+    }
+
+    private <C> NamespaceConfigBuilder<C> injectVaults(
+        NamespaceConfigBuilder<C> namespace)
+    {
+        if (isTlsEnabled)
+        {
+            namespace
+                .vault()
+                    .name("client")
+                    .type("filesystem")
+                    .options(FileSystemOptionsConfig::builder)
+                        .trust()
+                            .store("") // env
+                            .type("") // env
+                            .password("") // env
+                            .build()
+                        .build()
+                    .build()
+                .vault()
+                    .name("server")
+                    .type("filesystem")
+                    .options(FileSystemOptionsConfig::builder)
+                        .keys()
+                            .store("") // env
+                            .type("") // env
+                            .password("") //env
+                            .build()
+                        .build()
+                    .build();
+        }
+        return namespace;
+    }
+
+    private JsonPatch createEnvVarsPatch()
+    {
+        JsonPatchBuilder patch = Json.createPatchBuilder();
+        patch.replace("/bindings/tcp_client0/options/host", "${{env.TCP_CLIENT_HOST}}");
+        patch.replace("/bindings/tcp_client0/options/port", "${{env.TCP_CLIENT_PORT}}");
+
+        if (isJwtEnabled)
+        {
+            // jwt0 guard
+            patch.replace("/guards/jwt0/options/issuer", "${{env.JWT_ISSUER}}");
+            patch.replace("/guards/jwt0/options/audience", "${{env.JWT_AUDIENCE}}");
+            patch.replace("/guards/jwt0/options/keys/0/alg", "${{env.JWT_ALG}}");
+            patch.replace("/guards/jwt0/options/keys/0/kty", "${{env.JWT_KTY}}");
+            patch.replace("/guards/jwt0/options/keys/0/kid", "${{env.JWT_KID}}");
+            patch.replace("/guards/jwt0/options/keys/0/use", "${{env.JWT_USE}}");
+            patch.replace("/guards/jwt0/options/keys/0/n", "${{env.JWT_N}}");
+            patch.replace("/guards/jwt0/options/keys/0/e", "${{env.JWT_E}}");
+            patch.replace("/guards/jwt0/options/keys/0/crv", "${{env.JWT_CRV}}");
+            patch.replace("/guards/jwt0/options/keys/0/x", "${{env.JWT_X}}");
+            patch.replace("/guards/jwt0/options/keys/0/y", "${{env.JWT_Y}}");
+        }
```

Comment:
We don't want any environment vars in the composite namespaces.

Any parameterization should come from the `openapi` `server` or `client` binding, which can then of course use environment vars in `zilla.yaml` as desired.